### PR TITLE
Update WORKING-GROUPS.md to accurately reflect workgroup leads

### PR DESCRIPTION
**Pull request by brian-avery**
@howardjohn @costinm @ostromart @sdake @Monkeyanator Please take a look. I think this is accurate. This doc seems to have gotten quite out of date.

### CODE REVIEW

**Review by howardjohn — COMMENTED** [review]
LGTM except I am not sure about @ostromart 's role

**Review by ericvn — COMMENTED** [review]
LGTM - Picked up Jacob/Sam into Environments per 4/2 TOC Meeting notes. Rename/Recharter later per same meeting.

**Comment by ostromart** [comment]
LGTM, I've "moved on" from the lead role so this is accurate.

**Comment by hzxuzhonghu** [comment]
Suggest sorting them by WG

**Review by nrjpoddar — APPROVED** [review]

**Comment by sdake** [comment]
Hi gang. I miss the Istio team. That said, I approve of my removal for this
document.

I did not invest three years of my career in to this project. We often
spoke of an emeritus listing. I am not sure if this discussion turned into
a plan followed by action.

If not, @howardjohn would you be willing to advocate for such a document?
Determining the emeritus list should be a straightforward task with git.

Cheers
Steve

**Comment by nrjpoddar** [comment]
@sdake thanks for all your contributions! I did something similar for TOC and added an emeritus list and I think it's a great idea to do it for WG leads too.

**Comment by brian-avery** [comment]
Agreed. Thanks for your contributions @sdake . I didn't mean to belittle them, just wanted to update the leads list. Looking at https://github.com/istio/community/blob/master/ROLES.md, we have an Emeritus maintainer role. We should do similar for leads.

> If a maintainer becomes inactive in the project for an extended period of time, the individual will transition to being an emeritus maintainer. Emeritus maintainers lose their ability to approve code contributions, but retain their voting rights for up to one year. After one year, emeritus maintainers revert back to being normal members with no voting rights.

**Comment by ericvn** [comment]
So, when a Lead gives up that role (and it may just be to take another WG Lead or TOC position, or leave Istio all together), what should happen. I expect that they could still be a maintainer in that WG which gives them those rights in the former, or they probably don't need any rights if totally leaving Istio. Thus, my thought is that WG Lead emeritus can be acknowledged like TOC, but doesn't need rights for some length of time (if they stay a maintainer, they still have them). For maintainer, I'm fine changing it to a person not having rights from when they are removed. It will probably be some time before they would be removed after their last contribution (unless they specifically are asking to be removed at which point they probably don't need rights).

That leads to another question. What is the process by which WG leads should be cleaning up the maintainer lists for their WGs. I suspect that each Was list has people who haven't contributed in a while. Do we use eng.istio.io (which I think has when someone made an actual code contribution) as a start? Some might not make code contributions but still review docs. etc in the meetings so I wouldn't want to limit it to that. Maybe it's just a judgement call by the leads on each maintainer.

**Comment by nrjpoddar** [comment]
> Agreed. Thanks for your contributions @sdake . I didn't mean to belittle them, just wanted to update the leads list. Looking at https://github.com/istio/community/blob/master/ROLES.md, we have an Emeritus maintainer role. We should do similar for leads.
> 
> > If a maintainer becomes inactive in the project for an extended period of time, the individual will transition to being an emeritus maintainer. Emeritus maintainers lose their ability to approve code contributions, but retain their voting rights for up to one year. After one year, emeritus maintainers revert back to being normal members with no voting rights.
> 
> @nrjpoddar this seems different than how TOC defines emeritus. Should we change all three to have the same definition as TOC?

May be. I think we simply want a list of past WG leads to show our appreciation for their efforts. If usage of "emeritus" is confusing, we can just call it "Former WG Leads" and add folks. However, "Emeritus" WG Lead/TOC doesn't give you any privileges beyond an emeritus maintainer. Since these roles are sequential, a WG Lead/TOC member would already be a maintainer afaict.

tl; dr Emeritus maintainer should stay as is. You can add a "Emeritus WG Leads" which is just a list of former members without ascribing any specific privileges beyond emeritus maintainer.

**Comment by nrjpoddar** [comment]
@brian-avery "Working Group Leads:" team in org/teams.yaml should mirror the updated table.
